samples/highcharts/tooltip/formatter-extra-information/demo.js

```diff
@@ -0,0 +1,42 @@
+Highcharts.chart('container', {
+
+    tooltip: {
+        formatter: function () {
+            const basicInformation = 'The value for <b>' + this.x +
+            '</b> is <b>' + this.y + '</b>',
+                extraInformation = this.point.extraInfrmation || '';
+
+            return basicInformation + '<br>' + extraInformation;
+        }
+    },
+
+    xAxis: {
+        categories: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+    },
+
+    series: [{
+        data: [
+            29.9,
+            {
+                y: 71.5,
+                extraInfrmation: 'First month when XYZ introduced.'
```

Comment:
I think we should push the users to add additional info in a `data.custom` that was added specifically for cases like this
https://api.highcharts.com/highcharts/series.line.data.custom